syntax = "proto3";
package test_proto;

message SubData {
    string feedid = 1;
    int32 rank = 2;
    double score = 3;
}

message Data {
    int32 id = 1;
    string model_id = 2;
    SubData unit = 3;
}
